code/game/machinery/cryopod.dm

```diff
@@ -0,0 +1,455 @@
+		/obj/item/reagent_containers/hypospray/CMO,
+		/obj/item/clothing/accessory/medal/gold/captain,
+		/obj/item/clothing/gloves/krav_maga,
+		/obj/item/nullrod,
+		/obj/item/tank/jetpack,
+		/obj/item/documents,
+		/obj/item/nuke_core_container
+	)
+	// These items will NOT be preserved
+	var/static/list/do_not_preserve_items = list (
+		/obj/item/mmi/posibrain
+	)
+
+/obj/machinery/cryopod/Initialize()
+	..()
+	return INITIALIZE_HINT_LATELOAD //Gotta populate the cryopod computer GLOB first
+
+/obj/machinery/cryopod/LateInitialize()
+	update_icon()
+	find_control_computer()
+
+/obj/machinery/cryopod/proc/find_control_computer(urgent = 0)
+	for(var/M in GLOB.cryopod_computers)
+		var/obj/machinery/computer/cryopod/C = M
+		if(get_area(C) == get_area(src))
+			control_computer = C
+			break
+
+	// Don't send messages unless we *need* the computer, and less than five minutes have passed since last time we messaged
+	if(!control_computer && urgent && last_no_computer_message + 5*60*10 < world.time)
+		log_admin("Cryopod in [get_area(src)] could not find control computer!")
+		message_admins("Cryopod in [get_area(src)] could not find control computer!")
+		last_no_computer_message = world.time
+
+	return control_computer != null
+
+/obj/machinery/cryopod/close_machine(mob/user)
+	if(!control_computer)
+		find_control_computer(TRUE)
+	if((isnull(user) || istype(user)) && state_open && !panel_open)
+		..(user)
+		var/mob/living/mob_occupant = occupant
+		if(mob_occupant && mob_occupant.stat != DEAD)
+			to_chat(occupant, "<span class='boldnotice'>You feel cool air surround you. You go numb as your senses turn inward.</span>")
+		if(mob_occupant.client)//if they're logged in
+			despawn_world_time = world.time + (time_till_despawn * 0.1)
+		else
+			despawn_world_time = world.time + time_till_despawn
+	icon_state = "cryopod"
+
+/obj/machinery/cryopod/open_machine()
+	..()
+	icon_state = "cryopod-open"
+	density = TRUE
+	name = initial(name)
+
+/obj/machinery/cryopod/container_resist(mob/living/user)
+	visible_message("<span class='notice'>[occupant] emerges from [src]!</span>",
+		"<span class='notice'>You climb out of [src]!</span>")
+	open_machine()
+
+/obj/machinery/cryopod/relaymove(mob/user)
+	container_resist(user)
+
+/obj/machinery/cryopod/process()
+	if(!occupant)
+		return
+
+	var/mob/living/mob_occupant = occupant
+	if(mob_occupant)
+		// Eject dead people
+		if(mob_occupant.stat == DEAD)
+			open_machine()
+
+		if(!(world.time > despawn_world_time + 100))//+ 10 seconds
+			return
+
+		if(!mob_occupant.client && mob_occupant.stat < 2) //Occupant is living and has no client.
+			if(!control_computer)
+				find_control_computer(urgent = TRUE)//better hope you found it this time
+
+			despawn_occupant()
+
+// This function can not be undone; do not call this unless you are sure
+/obj/machinery/cryopod/proc/despawn_occupant()
+	var/mob/living/mob_occupant = occupant
+
+	if(istype(SSticker.mode, /datum/game_mode/cult))
```

Comment:
What if dynamic creates a cult?

Reply:
god i have no idea. this also needs to address changelings.

Reply:
Changelings updated but dynamic is another beast. Dynamic is so (dynamic) layered, it's hard to tell if there's an interaction. It'd only need to change the sac target into global